### DISCUSSION TITLE
Odroid or Celeron performance experience?

### DISCUSSION THREAD

**Discussion by chris948** [lloesche/valheim-server-docker#271] [Q&A] [answered]
I read through the discussions and issues, and the wiki and was kind of surprised at hardware requirements. Is the linux dedicated server version from Steam just inefficient? I'm just messing around on an ancient i5-3570 and running an empty dedicated server with Windows 10 and chrome browsers open, the entire machine averages 10% utilization. So I'm surprised at the Xeon E5-2620 30% at idle number. I would expect the two to be close. I do have a decent GTX 1650, but I have no idea if anything is being offloaded to it.

So the question, does anyone having experience running this on something like my odroid Celeron J4115? It would appear I don't have a chance comparing my celeron to the wiki's xeon 2620.

**Comment by lloesche** [comment, marked as answer]
https://www.cpubenchmark.net/cpu.php?cpu=Intel+Xeon+E5-2620+v3+%40+2.40GHz&id=2418
vs
https://www.cpubenchmark.net/cpu.php?cpu=Intel+Celeron+J4115+%40+1.80GHz&id=3679

**Reply by chris948** [reply]
yeah, thanks. I definitely did that and appreciate the incredibly well written documentation on this page. Like I said, what I don't get is why I'm getting so much better performance from my i5, so I was curious if there were more data points. All good.

https://www.cpubenchmark.net/cpu.php?cpu=Intel+Core+i5-3570K+%40+3.40GHz&id=828

Again, thanks for the awesome work.
